scripts/inner/kubestellar

```diff
@@ -192,11 +192,11 @@ function create_or_replace() { # usage: filename
 
 function ensure_root_compute_configd() {
```

Comment:
I know there was talk of doing this in a different PR, but I would find it easier to review _this_ PR if it simply removed this function.